pallets/file-system/src/utils.rs

```diff
@@ -587,18 +630,20 @@ where
             None
         };
 
-        let bsps_required = bsps_required.unwrap_or(ReplicationTarget::<T>::get());
+        let replication_target = replication_target.unwrap_or(T::DefaultReplicationTarget::get());
 
-        if bsps_required.is_zero() {
+        if replication_target.is_zero() {
             return Err(Error::<T>::ReplicationTargetCannotBeZero)?;
         }
 
-        if bsps_required > ReplicationTarget::<T>::get().into() {
-            return Err(Error::<T>::BspsRequiredExceedsTarget)?;
+        if replication_target > MaxReplicationTarget::<T>::get().into() {
+            return Err(Error::<T>::ReplicationTargetExceedsMaximum)?;
         }
 
         let current_tick =
             <T::ProofDealer as shp_traits::ProofsDealerInterface>::get_current_tick();
+
+        let zero = ReplicationTargetType::<T>::zero();
```

Comment:
just because I'm using it in 2 places
https://github.com/Moonsong-Labs/storage-hub/blob/bcd35d191b33b272fc386e889b31bdba2f5cf6c8/pallets/file-system/src/utils.rs#L657-L658